### DISCUSSION TITLE
Is it worthwhile spending time making checks for false conditions more consistent?

### DISCUSSION THREAD

**Discussion by AaronLi** [PlummersSoftwareLLC/NightDriverStrip#105] [General]
While looking through the code for drawing.cpp, I noticed that false conditions were checked 3 different ways in a window of 10 lines.

https://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/3a05a9b469106f148f9e6cd5bcb23b5ac6bf1166/src/drawing.cpp#L172-L182

We have `false == ...`, `... == false`, and `!...` all in one file. Does it make sense to spend time changing these to all one style such as using only `!` ?

Also see: `egrep  "([\w\W]+?( ?== ?false))|((false ?== ?)[\w\W]+?)" -Hrn`

**Comment by rbergen** [comment]
I would say it isn't worthwhile to apply consistency as a thing in itself. Restructuring Boolean logic requires extremely careful review, during which it's still easy to overlook unintended changes to the logic in question. As the modifications to the code should not lead to any change in behavior, I think the changes wouldn't do well on the "BlinkenPerBit" scale. 

If there is a reason to modify the code containing the logic for other reasons - or add new logic to it - then I would be supportive of applying a (more) consistent coding style.
